shared/chat/conversation/attachment-input/index.js

```diff
@@ -37,7 +38,29 @@ class RenderAttachmentInput extends Component<void, Props, State> {
     this.setState({title})
   }
 
+  // Temporary until we implement new mobile attachment design
+  renderMobile () {
+    const count = this.props.inputs.length
+    const currentTitle = this.props.inputs[this.state.index].title
+    return (
+      <PopupDialog onClose={this.props.onClose}>
```

Comment:
why not just make 2 style objects that override the margins using isMobile instead of a big copy/paste